From ff784f4803ab33f5e3389e40d038d52d1e211843 Mon Sep 17 00:00:00 2001
From: Yuan Chao <yuanc.fnst@cn.fujitsu.com>
Date: Wed, 28 Aug 2019 16:12:27 +0900
Subject: [PATCH] [PATCH] setup.py: remove the setup_requires for
 setuptools-scm

The setup_requires argument forces the download of the egg file for setuptools-scm
during the do_compile phase.  This download is incompatible with the typical fetch
and mirror structure.  The only usage of scm is the generation of the _version.py
file and in the release tarball it is already correctly created

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Derek Straka <derek@asterius.io>

Signed-off-by: Yuan Chao <yuanc.fnst@cn.fujitsu.com>

Rebase for pytest 6.1.0.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 setup.cfg | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 60f6564..c4d1471 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -55,7 +55,6 @@ package_dir =
 	=src
 setup_requires = 
 	setuptools>=40.0
-	setuptools-scm
 zip_safe = no
 
 [options.entry_points]
-- 
2.17.1

